build.gradle.kts

```
@@ -19,11 +19,11 @@ import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
 import java.net.URL
 
 plugins {
-  kotlin("jvm") version "1.6.0"
+  kotlin("jvm") version "1.6.10"
   `java-test-fixtures`
-  id("org.jetbrains.dokka") version "1.5.31"
-  id("com.google.devtools.ksp") version "1.6.0-1.0.1"
-  id("com.diffplug.spotless") version "6.0.0"
+  id("org.jetbrains.dokka") version "1.6.0"
```

Latest version of Dokka, no?
```suggestion
  id("org.jetbrains.dokka") version "1.6.10"
```